### ISSUE TITLE
Please document nsim property for libgloss nsim.specs. 

### ISSUE THREAD

**Issue by iguryanov** [foss-for-synopsys-dwc-arc-processors/toolchain#356]
Hi, 

please update hostlink page at https://foss-for-synopsys-dwc-arc-processors.github.io/toolchain/baremetal/hostlink.html 

It would be nice to have nsim option -prop=nsim_emt=1 mentioned near --specs=nsim.specs . 

Regards,
Igor.

**Comment by VVIsaev** [comment]
Should be fixed after #360. See https://foss-for-synopsys-dwc-arc-processors.github.io/toolchain/baremetal/hostlink.html  updated.
